utils/networks.ts

```diff
@@ -69,6 +71,9 @@ export const networks = {
         },
       ],
     },
+    tokenBlacklist: new Map([
+      ['0xc94dd466416A7dFE166aB2cF916D3875C049EBB7', '0xc94dd466416A7dFE166aB2cF916D3875C049EBB7'],
+    ]),
```

Comment:
This is temporary and we can handle differently once we have the token blacklist.

Reply:
#71

Reply:
You mean this choice of two tokens is temporary for testing right? If so maybe just add a `// TODO these are temporary, see https://github.com/ScopeLift/railgun-shielder/issues/71" style comment

Reply:
Yes, exactly will do

Reply:
And also, this is a single token